pkg/driverbuilder/builder/templates/ubuntu.sh

```diff
@@ -22,6 +22,7 @@ tar -xf data.tar.*
 {{end}}
 
 cd /tmp/kernel-download/usr/src/
+ls -altr
```

Comment:
I think i was the one removing all of these :/ but yep you are right, they help with debugging (even if they generate so much noice). It's ok to add it back!

Reply:
As long as it's not `-v` on `tar`, I'm good for more debug commands haha. The `tar -xvf` is the massive one...